systemvm/debian/opt/cloud/bin/setup/vpcrouter.sh

```diff
@@ -115,6 +115,8 @@ setup_vpcrouter() {
   sed -i "s#^OnCalendar=.*#OnCalendar=$LOGROTATE_FREQUENCY#g" /usr/lib/systemd/system/logrotate.timer
   sed -i 's#^AccuracySec=.*#AccuracySec=5m#g' /usr/lib/systemd/system/logrotate.timer
 
+  sed -i 's/^#\(dhcp-leasefile=\/var\/lib\/misc\/dnsmasq.leases\)/\1/' /etc/dnsmasq.conf
```

Comment:
just curious, this setting does not exist in VR of isolated networks as well

```
root@r-1171-VM:~# grep -r dnsmasq.leases /etc/dnsmasq*
/etc/dnsmasq.conf:#dhcp-leasefile=/var/lib/misc/dnsmasq.leases
/etc/dnsmasq.conf.tmpl:#dhcp-leasefile=/var/lib/misc/dnsmasq.leases
```

Does the issue exist with redundant VRs of isolated network as well ?

Reply:
good point @weizhouapache . Let's test that as well.

Reply:
good call @weizhouapache in the redundant isolated VR the issue remains. cc @Pearl1594

Reply:
added, but now I will have to look at shared networks as well, don't I?

Reply:
maybe not needed, as  redundant VRs are not supported

Reply:
this fails for isolated and shared network VRs :(

Reply:
note that after expunging the second VM both entries disappeared.